weatherWidget/styles/backgroundPic.css

```diff
@@ -0,0 +1,16 @@
+body {
+   background-image: url(https://www.woodsidespaandglamping.com/wp-content/uploads/2017/02/blurred-forest.jpg);
```

Comment:
I was talking about using the original background from mockup. Use it as is, do not add blurred effect :) No need to fix it here, just use this background in second homework.
![image](https://user-images.githubusercontent.com/71438780/94925882-9a157580-04c8-11eb-8425-4ee4c53ecb32.png)